xbmc/pvr/PVRManager.cpp

```diff
@@ -824,11 +824,16 @@ int CPVRManager::TranslateIntInfo(const CFileItem &item, DWORD dwInfo) const
   return IsStarted() && m_guiInfo ? m_guiInfo->TranslateIntInfo(item, dwInfo) : 0;
 }
 
-bool CPVRManager::GetVideoLabel(const CFileItem &item, int iLabel, std::string &strValue) const
+bool CPVRManager::GetVideoLabel(const CFileItem *item, int iLabel, std::string &strValue) const
```

Comment:
Any particular reason for changing this?

Reply:
Yes, this is a wrapper for CPVRGUIInfo::GetVideoLabel. The latter was cuffed to internally use CPVRItem which can only be created using a raw or shared CFileItem pointer. I decided not to change CPVRItem implementation ('some' effort) because CGUIInfomanager already has the item as raw pointer and to just use it ('less' effort).

Reply:
Okay